config/crd/bases/multiarch.openshift.io_clusterpodplacementconfigs.yaml

```
@@ -281,6 +337,62 @@ spec:
                     type: object
                 type: object
                 x-kubernetes-map-type: atomic
+              plugins:
+                description: |-
+                  Plugins selects the namespaces where the pod placement operand can process the nodeAffinityScoring
+                  of the pods. If left empty, all the namespaces are considered.
+                  The default sample allows to exclude all the namespaces where the
+                properties:
+                  nodeAffinityScoring:
+                    description: NodeAffinityScoring is the plugin that implements
+                      the ScorePlugin interface.
+                    properties:
+                      args:
+                        description: |-
+                          Args contains the configuration and is required.
+                          kubebuilder:"validation:Required"
+                        properties:
+                          platforms:
+                            description: Platforms is a required field and must contain
+                              at least one entry.
+                            items:
+                              description: PlatformConfig holds configuration for
+                                specific platforms, with required fields validated.
+                              properties:
+                                architecture:
+                                  description: |-
+                                    Architecture must be a non-empty string.
+                                    kubebuilder:"validation:Required"
+                                  type: string
+                                weight:
+                                  description: kubebuilder:"validation:Required,validation:Minimum:=0
```

```suggestion
                                  description: kubebuilder:"validation:Required,validation:Minimum:=0
```
This is not correct....

---

Left comments up above, so when you regen - it should fix